docs/source/installation/for_dev.rst

```diff
@@ -0,0 +1,143 @@
+Manim Installation For Developers
+=================================
+
+	This documentation is for developers who want to contribute to ``ManimCommunity/manim``.
+
+Installing Poetry
+*****************
+
+Poetry can be easily installed in any OS by just running the below command.
+
+If your system has the ``curl`` program installed, which is usually true in Linux and Mac,
+
+.. code-block:: bash
+	
+	  curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+
+
+If you are a Windows developer and want to use PowerShell, you can use the below command.
+
+.. code-block:: powershell
+	
+	  (Invoke-WebRequest -Uri https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py -UseBasicParsing).Content | python
+
+.. note:: Poetry can be installed by other ways also `see <https://python-poetry.org/docs/#alternative-installation-methods-not-recommended>`_.
```

Comment:
This renders as "... other ways also [see](https://python-poetry.org/docs/#alternative-installation-methods-not-recommended).", which is not ideal.

```suggestion
.. note:: Poetry can be installed by other ways also see `<https://python-poetry.org/docs/#alternative-installation-methods-not-recommended>`_.
```